tests/magma_linalg_test.py

```diff
@@ -114,5 +114,40 @@ def testEigMagmaConfig(self):
       hlo = jax.jit(partial(lax_linalg.eig, use_magma=True)).lower(a).as_text()
       self.assertIn('magma = "on"', hlo)
 
+  @jtu.sample_product(
+    shape=[(3, 4), (3, 3), (4, 3), (4, 3)],
+    dtype=float_types + complex_types,
+  )
+  @jtu.run_on_devices("gpu")
+  def testPivotedQrFactorization(self, shape, dtype):
+    if jtu.jaxlib_version() <= (0, 4, 38):
+      self.skipTest("qr with `pivoting=True` on GPU requires jaxlib version > 0.4.38")
+    if not gpu_solver.has_magma():
+      self.skipTest("MAGMA is not installed or can't be loaded.")
+    # TODO(b/377907938), TODO(danfm): Debug issues MAGMA support for
+    # complex128 in some configurations.
+    if dtype == np.complex128:
+      self.skipTest("MAGMA support for complex128 types is flaky.")
```

Comment:
Did you find this to be true? If not, maybe remove this, and I'm happy to disable if needed.

Reply:
I've just check and don't get this issue so have now removed this